srcpkgs/gala/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'gala'
+pkgname=gala
+version=3.2.0
+revision=1
+build_style=meson
+hostmakedepends="cmake pkg-config gettext glib-devel"
+makedepends="gnome-desktop-devel vala-devel bamf-devel plank-devel granite-devel gnome-settings-daemon-devel mutter-devel libcanberra-devel"
+short_desc="Gala Window Manager"
+maintainer="linarcx <linarcx@riseup.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/elementary/gala"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=137ceb10f9649defd30c27606fb63eb82c3622eb8aa45339747c135b03d723a8
+
+gala-devel_package() {
+	short_desc+=" - development files"
+	depends="gala"
```

Comment:
Typically a devel subpackage needs to depend on the makedepends of the parent package. Please just use the commit from #20429 as-is.

Reply:
right, and `depends="$pkgname"` is also wrong because xbps does not support such unversioned dependencies, i.e must be `depends="gala-${version}_${revision}"`